python/paddle/fluid/layers/nn.py

```diff
@@ -1012,7 +1012,9 @@ def cross_entropy(input, label, soft_label=False):
                                tensor<float/double> with shape [N x D].
         soft_label (bool): a flag indicating whether to
                                            interpretate the given labels as soft
-                                           labels, default `False`.
+                                           labels. Default: `False`.
+        ignore_index (int): Specifies a target value that is ignored and does not
+                            contribute to the input gradient. Default: -100
```

Comment:
Need to tell uers this argument is only worked when soft_label=False.

Reply:
Done